company/forms.py

```diff
@@ -37,6 +40,33 @@ class Meta:
         ]
 
 
+class MoveProductsForm(SingleButtonMixin, forms.Form):
+    new_company = forms.ModelChoiceField(
```

Comment:
Wydaje mi się, że warto by dodać nowe pole source_company z widgetem typu hidden, aby jawnie zdefiniować wszystkie parametry wejściowe. To nam zweryfikuje, czy np. odwołujemy się do istniejacego obiektu.